index.ts

```diff
@@ -1,46 +1,58 @@
-function detect() {
+export interface Version {
+  name?: string
+  version?: string
+  os?: string
+  bot?: boolean
+};
+
+type Nullable<T> = T | null;
```

Comment:
Agree that I should have something like this in #79 (or possibly `Maybe<T>` - mainly because I like that functional naming better; but that's just personal preference).